scrapy_dropbox/__init__.py

```diff
@@ -1,46 +1,98 @@
-import dropbox
-import typing
+import logging
 import os
 
+from dropbox import Dropbox, files
+from dropbox.exceptions import ApiError
+from dropbox.files import WriteMode
 from scrapy.exceptions import NotConfigured
 from scrapy.extensions.feedexport import BlockingFeedStorage
-from dropbox.files import WriteMode
+
+from .utils import parse_uri
+
+logger = logging.getLogger(__name__)
+CHUNK_SIZE = 4 * 1024 * 1024
```

Comment:
Maybe you could define this value in the class, so that subclasses may override it easily?